src/Metadata/Builder/ColumnBuilder.php

```diff
@@ -98,7 +98,7 @@ public function setName(string $name): self
         $name = trim($name);
 
         if (empty($name)) {
-            throw new InvalidArgumentException('Table\'s name cannot be empty.');
+            throw new InvalidArgumentException('Column\'s name cannot be empty.');
```

Comment:
V `mssql` je dovolene aby bol nazov stlpca zlozeny iba z medzier, ... fixnem to v `ex-mssql` ... no zistil som, ze tu je zla chybova hlaska, ktora ma dost zmiatla pri debugovaní problému.